recipe/meta.yaml

```diff
@@ -291,7 +292,8 @@ outputs:
         - {{ pin_subpackage('tensorflow-estimator', exact=True) }}
         # avoid that people without GPUs needlessly download ~0.5-1GB
         # This also helps mamba give preference to the CPU build
-        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version == "11.8"]
```

Comment:
should cuda version just be added to the `host` section instead of manually adding it to the `run` section?